src/app/persons/persons.component.html

```diff
@@ -0,0 +1,6 @@
+<nz-layout>
+    <nz-content>
+        <h1>Work experience of {{editedPerson?.name}} {{editedPerson?.surname}}</h1>
+        <app-experience-form [person]="editedPerson"></app-experience-form>
+    </nz-content>
```

Comment:
No here goes only:

```html
<router-outlet></router-outlet>
```

What if we add rates edit in this module or move person edit there ...